snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/io/SnomedExportApiTest.java

```diff
@@ -1230,6 +1232,59 @@ public void exportUnpublishedOWLExpressionRefsetMembers() throws Exception {
 		assertArchiveContainsLines(exportArchive, fileToLinesMap);
 	}
 	
+	@Test
+	public void exportRf2WithConfiguration() throws Exception {
+		
+		final Map<String, Object> codeSystemExportSettings = Map.of(
+				"maintainerType",  Rf2MaintainerType.NRC,
+				"nrcCountryCode", "GB",
+				"extensionNamespaceId", "370137002",
+				"refSetLayout", Rf2RefSetExportLayout.COMBINED
+				);
+		
+		String codeSystemId = "SNOMEDCT-custom-rf2-export-config";
+		createCodeSystem(null, branchPath.getPath(), codeSystemId, codeSystemExportSettings).statusCode(201);
+		
+		final Map<String, Object> config = Map.of(
+			"type", Rf2ReleaseType.SNAPSHOT.name(),
+			"includeUnpublished", true,
+			"codeSystemId", codeSystemId
+		);
+		
+		final File exportArchive = doExport(codeSystemId, config);
+		
+		final Map<String, Boolean> files = ImmutableMap.<String, Boolean>builder()
+				.put("der2_sssssssRefset_MRCMDomainSnapshot_GB", true)
+				.put("sct2_Relationship_Snapshot_GB", true)
+				.put("sct2_Description_Snapshot-en_GB", true)
+				.put("der2_sRefset_SimpleMapSnapshot_GB", true)
+				.build();
+			
+		assertArchiveContainsFiles(exportArchive, files);
+		
+	}
+	
+	@Test
+	public void exportRf2WithoutConfiguration() throws Exception {
+		
+		String codeSystemId = "SNOMEDCT-default-rf2-export-config";
+		createCodeSystem(branchPath, codeSystemId).statusCode(201);
+		
+		CodeSystem codeSystem = CodeSystemRestRequests.getCodeSystem(codeSystemId).extract().as(CodeSystem.class);
+		
+		final File exportArchive = doExport(codeSystemId, codeSystem.getSettings());
+		
+		final Map<String, Boolean> files = ImmutableMap.<String, Boolean>builder()
+				.put("der2_sssssssRefset_MRCMDomainSnapshot_INT", true)
+				.put("sct2_Relationship_Snapshot_INT", true)
+				.put("sct2_Description_Snapshot-en_INT", true)
+				.put("der2_sRefset_SimpleMapSnapshot_INT", true)
+				.build();
+			
+		assertArchiveContainsFiles(exportArchive, files);
+		
+	}
```

Comment:
It would be nice to have additional tests that verify different export configuration, like defining both namespace, maintainerType and other properties and asserting the end file name got computed properly or not.